pom.xml

```diff
@@ -39,6 +39,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>io.projectreactor</groupId>
+                <artifactId>reactor-bom</artifactId>
+                <version>${reactor}</version>
```

Comment:
The import is already present on the Build Parent level, see https://github.com/spring-projects/spring-data-build/blob/5fec771ae2c33ea71fbfc0767ace95d7fbba304b/parent/pom.xml#L845

Reply:
Interesting. I wonder why the build was still using 3.4.6 then. Is the test-scoped `reactor-test` dependency below with a specific version enough to override the version of `reactor-core` in the compile scope to 3.4.6 too?

Reply:
Aah, it wasn't. CI's been broken since the start of the week: https://jenkins.spring.io/view/SpringData/job/spring-data-couchbase/job/main/626/console.